js/search/searchConfig.js

```diff
@@ -26,12 +26,27 @@ const libraryPath = isMac ? path.join(macLibraryPath, 'libsymphonysearch.dylib')
 const userConfigFileName = 'search_users_config.json';
 const userConfigFile = isDevEnv ? path.join(__dirname, '..', '..', userConfigFileName) : path.join(userData, userConfigFileName);
 
+const libraryFolderPath = isMac ? macLibraryPath : winLibraryPath;
+
+const launchAgentFile = path.join(libraryFolderPath, 'search-launch-agent.sh');
+const launchDaemonFile = path.join(libraryFolderPath, 'search-launch-daemon.sh');
+const windowsTaskFile = path.join(libraryFolderPath, 'search-win-task.exe');
+const windowsClearScript = path.join(libraryFolderPath, 'clear-on-launch.exe');
+
+const scriptExt = isMac ? '.sh' : '.exe';
```

Comment:
It's a little dangerous to assume that it has to be an "exe" extension if it is not a Mac. What if we decide to support Linux in the future?

Reply:
👍